tests/test1016.py

```diff
@@ -0,0 +1,54 @@
+#Needs: fonts/feta20.pfb
+
+# feta is a font for music symbols, so reencode
+# to ucs4 means no encoding slots are used; but
+# you can still select by the symbol names.
```

Comment:
The font itself being used for these tests doesn't matter, right? So let's just say that these are tests for `isWorthOutputting` and we're using a music symbol font. This information isn't really necessary to understand the tests.

Reply:
A symbol font means you can't select by unicode value. The information is relevant to understand what the test does and does not do (unless you are asking to rewrite the test with a text font).